apps/src/lab2/levelEditors/aiCustomizations/UpdateContext.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  AiCustomizations,
+  LevelAiCustomizations,
+  ModelCardInfo,
+  Visibility,
+} from '@cdo/apps/aichat/types';
+import {createContext} from 'react';
+
+export const UpdateContext = createContext({
+  aiCustomizations: {} as LevelAiCustomizations,
+  setPropertyValue: (
```

Comment:
Oh sure, this is similar to setting up a redux store/actions (in that there is a single state value, aiCustomizations, and some action functions to update it) - but without actually doing all the redux boilerplate because this is a relatively small surface area. The main benefit here is just that we don't need to thread the `aiCustomizations` value and `setXYZ` functions through multiple layers of props.